Update uninstall and upgrade collector for Windows (OTel) (#6616)
SumoLogic/sumologic-documentation: blog-collector/2026-04-30-otel.md

---
title: Version 0.151.0 (OpenTelemetry Collector)
image: https://assets-www.sumologic.com/company-logos/_800x418_crop_center-center_82_none/SumoLogic_Preview_600x600.jpg?mtime=1617040082
keywords:
  - opentelemetry
  - collection
hide_table_of_contents: true    
---

> In this release, we've added **Winget** (Windows Package Manager) support for uninstalling and upgrading our OpenTelemetry Collector on Windows using install scripts. This improves deployment flexibility, streamlines upgrades on supported systems, and maintains compatibility with existing installation methods. [Learn more](/docs/send-data/opentelemetry-collector/install-collector/windows/#uninstall).

Reviewer: ```suggestion
In this release, we've added **Winget** (Windows Package Manager) support for uninstalling and upgrading the OpenTelemetry Collector on Windows. This streamlines lifecycle management on supported systems while maintaining full compatibility with existing installation methods. [Learn more](/docs/send-data/opentelemetry-collector/install-collector/windows/#uninstall).
```
